tests/ui/sanitize/sanitizer-dataflow-abilist-rust-no-core-no-std.txt

```diff
@@ -0,0 +1,13 @@
+fun:__rdl_alloc=uninstrumented
+fun:__rdl_alloc_zeroed=uninstrumented
+fun:__rdl_dealloc=uninstrumented
+fun:__rdl_realloc=uninstrumented
+fun:__rg_oom=uninstrumented
+fun:__rust_alloc=uninstrumented
+fun:__rust_alloc_error_handler=uninstrumented
+fun:__rust_alloc_zeroed=uninstrumented
+fun:__rust_dealloc=uninstrumented
+fun:__rust_realloc=uninstrumented
+fun:_ZN4core*=uninstrumented
+fun:_ZN3std*=uninstrumented
```

Comment:
Do we need separate abilist files for no-core / no-std?

I would have thought core/std rust is a superset of no-core/no-std rust?

If so, I think we can have one ABI list for core/std configuration, then building no-core/no-std should just not hit any cases where we need to apply the abilist rules for symbols beyond no-core/no-std.

Reply:
Sorry, by no core and no std in the file name it means core and std are uninstrumented since we don't build tests with Cargo and thus can't use the Cargo build-std feature, not that these lists are for use with the `no_core` and `no_std` features. I also moved these ABI list files to test/sanitizer so it implies they should only beused by these tests. The actual ABI list files users should use are shipped with the 'sanitizers` crate.

Reply:
I also removed them and consolidated the ABI list files into a single ABI list file for the tests without the no core and no std in the file name.